test/e2e/sql/src/test/resources/env/scenario/db_tbl_sql_federation/proxy/conf/mysql/config-dbtbl-with-sql-federation.yaml

```diff
@@ -0,0 +1,167 @@
+    username: test_user
+    password: Test@123
+    connectionTimeoutMilliseconds: 30000
+    idleTimeoutMilliseconds: 60000
+    maxLifetimeMilliseconds: 1800000
+    maxPoolSize: 50
+    minPoolSize: 2
+  sql_federation_ds_5:
+    url: jdbc:mysql://mysql.db_tbl_sql_federation.host:3306/sql_federation_ds_5?useSSL=false&characterEncoding=utf-8
+    username: test_user
+    password: Test@123
+    connectionTimeoutMilliseconds: 30000
+    idleTimeoutMilliseconds: 60000
+    maxLifetimeMilliseconds: 1800000
+    maxPoolSize: 50
+    minPoolSize: 2
+  sql_federation_ds_6:
+    url: jdbc:mysql://mysql.db_tbl_sql_federation.host:3306/sql_federation_ds_6?useSSL=false&characterEncoding=utf-8
+    username: test_user
+    password: Test@123
+    connectionTimeoutMilliseconds: 30000
+    idleTimeoutMilliseconds: 60000
+    maxLifetimeMilliseconds: 1800000
+    maxPoolSize: 50
+    minPoolSize: 2
+  sql_federation_ds_7:
+    url: jdbc:mysql://mysql.db_tbl_sql_federation.host:3306/sql_federation_ds_7?useSSL=false&characterEncoding=utf-8
+    username: test_user
+    password: Test@123
+    connectionTimeoutMilliseconds: 30000
+    idleTimeoutMilliseconds: 60000
+    maxLifetimeMilliseconds: 1800000
+    maxPoolSize: 50
+    minPoolSize: 2
+  sql_federation_ds_8:
+    url: jdbc:mysql://mysql.db_tbl_sql_federation.host:3306/sql_federation_ds_8?useSSL=false&characterEncoding=utf-8
+    username: test_user
+    password: Test@123
+    connectionTimeoutMilliseconds: 30000
+    idleTimeoutMilliseconds: 60000
+    maxLifetimeMilliseconds: 1800000
+    maxPoolSize: 50
+    minPoolSize: 2
+  sql_federation_ds_9:
+    url: jdbc:mysql://mysql.db_tbl_sql_federation.host:3306/sql_federation_ds_9?useSSL=false&characterEncoding=utf-8
+    username: test_user
+    password: Test@123
+    connectionTimeoutMilliseconds: 30000
+    idleTimeoutMilliseconds: 60000
+    maxLifetimeMilliseconds: 1800000
+    maxPoolSize: 50
+    minPoolSize: 2
+
+rules:
+- !SINGLE
+  tables:
+    - "*.*"
+- !SHARDING
+  tables:
+    t_order:
+      actualDataNodes: sql_federation_ds_${0..9}.t_order_${0..9}
+      databaseStrategy:
+        standard:
+          shardingColumn: user_id
+          shardingAlgorithmName: it_standard_fixture
+      tableStrategy:
+        standard:
+          shardingColumn: order_id
+          shardingAlgorithmName: it_standard_fixture
+    t_order_item:
+      actualDataNodes: sql_federation_ds_${0..9}.t_order_item_${0..9}
+      databaseStrategy:
+        standard:
+          shardingColumn: user_id
+          shardingAlgorithmName: it_standard_fixture
+      tableStrategy:
+        standard:
+          shardingColumn: order_id
+          shardingAlgorithmName: it_standard_fixture
+      keyGenerateStrategy:
+        column: item_id
+        keyGeneratorName: auto_increment
+      auditStrategy:
+        auditorNames:
+          - auditor_constant
+        allowHintDisable: true
+    t_order_details:
```

Comment:
Please remove these non-standard tables.  Refer this doc - https://github.com/apache/shardingsphere/issues/21286